app/main.py

```diff
@@ -1,8 +1,39 @@
+from __future__ import annotations
+
+
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+
+def find_partner(person: Person, spouse_name: str, is_wife: bool) -> None:
+    spouse = Person.people.get(spouse_name)
+    if spouse:
+        if is_wife:
+            person.wife = spouse
+            spouse.husband = person
+        else:
+            person.husband = spouse
+            spouse.wife = person
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    created_person_list = \
+        [
+            Person(persona["name"], persona["age"])
+            for persona in people
+        ]
```

Comment:
Do not make line breaking on this line. Avoid using `\`, place `[` after `=` instead:
```suggestion
    created_person_list = [
        Person(persona["name"], persona["age"])
        for persona in people
    ]
```